Source/Rows/TextAreaRow.swift

```diff
@@ -80,6 +80,7 @@ open class _TextAreaCell<T> : Cell<T>, UITextViewDelegate, AreaCell where T: Equ
         textView.font = .preferredFont(forTextStyle: .body)
         textView.textContainer.lineFragmentPadding = 0
         textView.textContainerInset = UIEdgeInsets.zero
+        textView.backgroundColor = .clear
         contentView.addSubview(textView)
 
         let placeholderLabel = UILabel()
```

Comment:
The `placeholderLabel.textColor` must also be updated. It does not have to be the same color. There may be a suitable existing dynamic color

Reply:
Done in the latest commit.